{"name": "test_create_order_without_auth_without_ingredients_error", "status": "passed", "description": "Проверка создания заказа без ингредиентов не авторизованным пользователем. Получаем код 400 и тело ответа в котором есть: 'success':false", "steps": [{"name": "Создание заказа", "status": "passed", "parameters": [{"name": "token_user", "value": "''"}, {"name": "ids", "value": "''"}], "start": 1717267064322, "stop": 1717267064532}], "start": 1717267064321, "stop": 1717267064533, "uuid": "cc54609f-732b-4324-8c8f-5e5c22613049", "historyId": "8c98ccb552b3074159d985d80502a3ac", "testCaseId": "8c98ccb552b3074159d985d80502a3ac", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_without_auth_without_ingredients_error", "labels": [{"name": "story", "value": "Ручка /api/orders"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "WIN-BHU1OVNKOHE"}, {"name": "thread", "value": "13424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}